projects/plugins/crm/composer.json

```diff
@@ -10,8 +10,8 @@
 		"codeception/module-db": "^1.0",
 		"codeception/module-filesystem": "^1.0",
 		"codeception/util-universalframework": "^1.0",
-		"automattic/wordbless": "0.4.2",
-		"yoast/phpunit-polyfills": "1.0.4"
+		"yoast/phpunit-polyfills": "1.0.4",
+		"symfony/polyfill-php81": "^1.27"
```

Comment:
We needed to add `symfony/polyfill-php81` as a dev dependency to be able to use `WP_UnitTestCase`.

The reason is that `enum_exists` did not exist in the test environment which made mocking fail.

Reply:
It didn't really leave a trail, so I've had to go by assumptions, but the only places `enum_exists` is being used are `doctrine/instantiator` and `myclabs/deep-copy`. Both of them are dependencies of phpunit

Reply:
`yoast/phpunit-polyfills` should have chosen an appropriate version of phpunit for the version of PHP installed. Did that not happen?

Reply:
It didn't appear to do so, no. At least locally but I don't see why it would be different in GitHub

Reply:
Let's give it a try anyway, I think it really will work without it in GitHub.

Reply:
Incoming!

Reply:
Just sharing what I got locally for context:

```
There was 1 error:

1) Automattic\Jetpack\CRM\Tests\REST_Contacts_Controller_Test::test_get_item_unknown_fatal_error
Error: Call to undefined function enum_exists()

/usr/local/src/jetpack-monorepo/projects/plugins/crm/tests/php/rest-api/v4/class-rest-contacts-controller-test.php:85
```

Reply:
It seems you were correct in your assumption @anomiex 👍 
Then I just have to figure out why my own Jetpack Monorepo environment is complaining 😱

Reply:
I update and recreated my Docker container and the issue went away; but I can't say if there's an underlying issue since what I've done is go from PHP v8.0.28 to v8.2.7 and `enum_exists` is natively supported in v8.1+

Guess: some fringe version check that failed with 8.0.28 in `yoast/phpunit-polyfills`?

That being said: GitHub is happy and others on my team can successfully build, so I'm calling it a resolved ✅